router/src/main/java/flow/StateSizeLogger.kt

```diff
@@ -0,0 +1,23 @@
+package flow
```

Comment:
This doesn't need to be in the same package. As long as you are accessing the private property after setting the `isAccessible` property on the `Field`. It doesn't really matter anyways.

Reply:
Makes sense.